packages/renderer/src/lib/events/EventUI.ts

```diff
@@ -0,0 +1,29 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+//import type { V1ObjectMeta } from '@kubernetes/client-node';
+
+export interface EventUI {
+  count?: number;
```

Comment:
we should probably have the uid ?

Reply:
I don't think so. UID has been added to the Deployment because they can be referenced by Events using this UID. But here, an event won't be referenced by another event.